### PR TITLE
fixes #2912 add dorsal iris

### LINKED ISSUE #2912: dorsal iris
**Preferred term label:** dorsal iris

**Synonyms** dorsal part of iris https://orcid.org/0000-0002-0819-0473

**Definition (free text, please give PubMed ID)** 
The dorsal part of the iris. This region is involved in lens regeneration in urodeles (newts and salamanders) in a process named Wolffian lens regeneration. In these species, lens regeneration proceeds through two steps. The first step involves the entire circumference of the iris pigmented epithelium. The second step is confined to the pigmented epithelial cells located along the dorsal rim of the iris and involves dorsal-specific activation of the Wnt signaling pathway. (PMID: 17030116) (PMID: 30986388) (PMID: 29579253)

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
multi-tissue structure UBERON:0000481
iris UBERON:0001769

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2912

### CODE REVIEW

**Comment by cmungall** [comment]
This should be failing qc checks due to missing isa

**Comment by anitacaron** [comment]
@cmungall, checking the `profile.txt`, the level of QC for `missing_superclass` is INFO.

**Comment by anitacaron** [comment]
#gogoeditdiff

**Comment by matentzn** [comment]
robot report MISSING_SUPERCLASS is too general. We would need something more bespoke, like "if it starts with UBERON: it needs to be an UBERON:anatomical entity" SPARQL thing.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224055,6 +224055,14 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0006767 ! head of femur
 property_value: dcterms-date "2023-05-30T06:34:18Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480063
+name: dorsal iris
+def: "The dorsal part of the iris. This region is involved in lens regeneration in urodeles (newts and salamanders) in a process named Wolffian lens regeneration. In these species, lens regeneration proceeds through two steps. The first step involves the entire circumference of the iris pigmented epithelium. The second step is confined to the pigmented epithelial cells located along the dorsal rim of the iris and involves dorsal-specific activation of the Wnt signaling pathway." [PMID:17030116, PMID:29579253]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224061]: Consider keeping the first two sentences in the text definition and relegating the rest to a comment.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224055,6 +224055,14 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0006767 ! head of femur
 property_value: dcterms-date "2023-05-30T06:34:18Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480063
+name: dorsal iris
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224060]: If "dorsal rim of the iris" is an exact synonym, consider adding (dbxref [PMID:29579253](https://pubmed.ncbi.nlm.nih.gov/29579253)).

**Comment by bvarner-ebi** [comment]
> This should be failing qc checks due to missing isa

Since 'anatomical entity' (CARO:0000000) is inferred, should this still fail?

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224061]: Also consider if a new GO term 'Wolffian lens regeneration' is warranted.

**Comment by paulacalipho** [comment]
@bvarner-ebi thanks for the input. I will make the changes in the definition and add the synonym. Should I try is_a "anatomical entity"?

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi thanks for the input. I will make the changes in the definition and add the synonym. Should I try is_a "anatomical entity"?

'anatomical entity' (CARO:0000000) is already inferred, so I don't think it needs to be manually asserted. I think adding superclass 'anatomical structure' would be reasonable.

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi I should add is_a organ part UBERON:0000064 instead of anatomical structure?

Uberon considers eye to be a 'sense organ', which is an organ, and the text definition of 'organ part' is "A multicellular structure that is a part of an organ", so it seems reasonable.

Also consider if adding 'has part' some 'iris epithelium' is appropriate.

**Comment by paulacalipho** [comment]
@cmungall @bvarner-ebi I don't think that there are parts of the dorsal iris epithelium that lies outside the dorsal iris

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224061]: Indeed this term does not exist in GO. Should I make a NTR to GO?

**Comment by bvarner-ebi** [comment]
> @cmungall @bvarner-ebi I don't think that there are parts of the dorsal iris epithelium that lie outside the dorsal iris

@paulacalipho, if there is 'iris epithelium' that exists outside the 'dorsal iris', overlaps would be appropriate.

**Comment by bvarner-ebi** [comment]
@matentzn, can you kindly advise- did QC fail due to memory constraints? Any advice on how to proceed?

```
HTTP request sent, awaiting response... 200 OK
Length: 97201 (95K) [text/plain]
Saving to: ‘tmp/GO.xrf_abbs’

     0K .......... .......... .......... .......... .......... 52% 9.07M 0s
    50K .......... .......... .......... .......... ....      100% 9.19M=0.01s

2023-06-15 11:51:36 (9.12 MB/s) - ‘tmp/GO.xrf_abbs’ saved [97201/97201]

Replaced: 0
Killed
make: *** [uberon.Makefile:1136: reports/basic-allcycles] Error 137
```

**Comment by anitacaron** [comment]
@bvarner-ebi, Sometimes, if you run again it works :)

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**